web_src/css/repo.css

```diff
@@ -3083,46 +3083,86 @@ tbody.commit-list {
   }
 }
 
-.pr-status {
+.ui.segment.commit-status-list {
+  padding: 0;
+  max-height: 30vh;
```

Comment:
Can we have a static pixel height like 300px or something? vh will not look good on small heights.

Reply:
FWIW, this is `max-height: 231px` on GitHub.

Reply:
Done in https://github.com/go-gitea/gitea/pull/26284/commits/08b7951f48dc7d1d8ac8413491a512ec4fa9a459